interfaces/data/IRss.ts

```diff
@@ -4,10 +4,16 @@
 
 import Parser from 'rss-parser';
 import { IRssItem } from './IRssItem';
+import { IRssPodcastValue } from './IRssPodcast';
```

Comment:
We added new interfaces for the `podcast:value` and `podcast:valueRecipient` tags